### DISCUSSION TITLE
重构 GitHub Actions Build

### DISCUSSION THREAD

**Discussion by allMagicNB** [Meloong-Git/PCL#3987] [讨论]
打算重构下现在的 Build，主要更改功能：
- [x] 上传压缩包命名增加 Commit Hash
- [x] 支持替换 Secret
- [ ] ……

你可以在下面 Comment 想增加的功能，看到了我觉得能做的话会加进来……

**Comment by Silverteal** [comment]
~~给编译产物进行upx加壳~~

**Comment by Deep-Dark-Forest** [comment]
Commit Hash 是取一部分还是完整的？

**Reply by zkitefly** [reply]
完整的或者前七个字符都可以

**Reply by allMagicNB** [reply]
一般来说前七位已经具有唯一性

**Comment by allMagicNB** [comment]
妈的，搞到一半浏览器炸了，刚才搞的全没了，现在又得重新搞

**Reply by hejiehao** [reply]
我选择
![image](https://github.com/Hex-Dragon/PCL2/assets/98140773/ea8f751c-52ec-4de2-a787-8c8224aff46f)
